.github/docker-node/10-amazonlinux2.Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM amazonlinux:2
+
+ENV NODE_VERSION 10.24.1
+
+# centos needs the user to be root; sudo doesn't work.
+USER root
```

Comment:
Why do we need the explicit setting to `root` here, by default the commands are run as the root user. Same applies for the other Dockerfiles.

Reply:
It is a remnant from pre-refactor. Based on test run it can be trimmed. 
(note: we run as root when using image inside container in workflows)